docs/README.md

```diff
@@ -598,7 +598,14 @@ GET /../../etc/password HTTP/1.1
 
 There are a few different *request item* types that provide a convenient mechanism for specifying HTTP headers, simple JSON and form data, files, and URL parameters.
 
-They are key/value pairs specified after the URL. All have in common that they become part of the actual request that is sent and that their type is distinguished only by the separator used: `:`, `=`, `:=`, `==`, `@`, `=@`, `:=@` and `==@`. The ones with an `@` expect a file path as value.
+They are key/value pairs specified after the URL. All have in common that
+they become part of the actual request that is sent and that their type is
+distinguished only by the separator used: `:`, `=`, `:=`, `==`. If you want
+to use a file's contents with these operators, you can simply add an `AT` (`@`)
+operator as the suffix. Which is going to make them  read the file's contents,
+and use it as the value for the specified request item (`:` and `==` will strip
+any trailing newlines automatically). The `AT` (`@`) operator itself can also be
+individually used to embed arbitrary data directly into the request's body.
```

Comment:
This paragraph introduces a fundamental concept and currently, it feels rather dense. Please continue working on it to make it more digestible for people just getting started with HTTPie. And the table below is inconsistent as it shows the `@` suffix for one of the types only. 

Also, the lack of examples makes it very abstract. I think we should add a comprehensive example similar to the JSON one:

```
$ http PUT pie.dev/put \
    name=John \                        # String (default)
    age:=29 \                          # Raw JSON — Number
    married:=false \                   # Raw JSON — Boolean
    hobbies:='["http", "pies"]' \      # Raw JSON — Array
    favorite:='{"tool": "HTTPie"}' \   # Raw JSON — Object
    bookmarks:=@files/data.json \      # Embed JSON file
    description=@files/text.txt        # Embed text file
```

Maybe a bigger refactoring is needed.

Reply:
Now that we’br had added the `@` suffix to headers and params, we should mention them in their respective sections as well (with a backlink to the request items section).